packages/gatsby-plugin-page-creator/src/path-utils.ts

```diff
@@ -1,7 +1,9 @@
+const extractModelRegex = /\{([a-zA-Z]+)\./
```

Comment:
Just checking if it's not something like this (numbers, underscores?)

```suggestion
const extractModelRegex = /\{([\w\d_]+)\./
```